app/main.py

```diff
@@ -1,6 +1,21 @@
-# write your imports here
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(
+    customers: list[Customer],
+    hall_number: int,
+    cleaner: Cleaner,
+    movie: str
+) -> None:
+    customer_objects = [Customer(cs["name"], cs["food"]) for cs in customers]
```

Comment:
```suggestion
    customer_objects = [Customer(cs["name"], cs["food"]) for cs in customers]
```
`cs` is a bad variable name.